docsrc/tutorials/deploy_torch_tensorrt_to_triton.rst

```diff
@@ -0,0 +1,212 @@
+for more details. 
+
+With the model repository setup, we can proceed to launch the Triton server
+with the docker command below.
+
+::
+
+   # Make sure that the TensorRT version in the Triton container
+   # and TensorRT version in the environment used to optimize the model
+   # are the same.
+
+   docker run --gpus all --rm -p 8000:8000 -p 8001:8001 -p 8002:8002 -v /full/path/to/docs/examples/model_repository:/models nvcr.io/nvidia/tritonserver:<xx.yy>-py3 tritonserver --model-repository=/models
+
+This should spin up a Triton Inference server. Next step, building a simple
+http client to query the server.
+
+Step 3: Building a Triton Client to Query the Server
+----------------------------------------------------
+
+Before proceeding, make sure to have a sample image on hand. If you don't
+have one, download an example image to test inference. In this section, we 
+will be going over a very basic client. For a variety of more fleshed out
+examples, refer to the `Triton Client Repository <https://github.com/triton-inference-server/client/tree/main/src/python/examples>`__
+
+::
+
+   wget  -O img1.jpg "https://www.hakaimagazine.com/wp-content/uploads/header-gulf-birds.jpg"
+
+We then need to install dependencies for building a python client. These will 
+change from client to client. For a full list of all languages supported by Triton,
+please refer to `Triton's client repository <https://github.com/triton-inference-server/client>`__.
+
+::
+
+   pip install torchvision
+   pip install attrdict
+   pip install nvidia-pyindex
+   pip install tritonclient[all]
+
+Let's jump into the client. Firstly, we write a small preprocessing function to
+resize and normalize the query image.
+
+::
+
+   import numpy as np
+   from torchvision import transforms
+   from PIL import Image
+   import tritonclient.http as httpclient
+   from tritonclient.utils import triton_to_np_dtype
+
+   # preprocessing function
+   def rn50_preprocess(img_path="img1.jpg"):
+       img = Image.open(img_path)
+       preprocess = transforms.Compose([
+           transforms.Resize(256),
+           transforms.CenterCrop(224),
+           transforms.ToTensor(),
+           transforms.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225]),
+       ])
+       return preprocess(img).numpy()
+
+   transformed_img = rn50_preprocess()
+
+Building a client requires three basic points. Firstly, we setup a connection
+with the Triton Inference Server.
+
+::
+
+   # Setting up client
+   triton_client = httpclient.InferenceServerClient(url="localhost:8000")
+
+Secondly, we specify the names of the input and output layer(s) of our model.
+
+::
+
+   test_input = httpclient.InferInput("input__0", transformed_img.shape, datatype="FP32")
+   test_input.set_data_from_numpy(transformed_img, binary_data=True)
+
+   test_output = httpclient.InferRequestedOutput("output__0", binary_data=True, class_count=1000)
```

Comment:
Should it be `triton_client` here instead of `httpclient` ?

Reply:
or triton_client is only used during triton_client.infer() ?

Reply:
bad naming on my part, triton_client is only used using infer(). Changing var names